openstack_controller/datadog_checks/openstack_controller/data/conf.yaml.example

```diff
@@ -85,6 +85,19 @@ instances:
         domain:
             id: <DOMAIN_ID>
 
+  ## @param openstack_config_file_path - optional - string
+  ## Absolute path of the configuration file for the connection to openstack with openstacksdk.
+  #
+  # openstack_config_file_path: "/path/to/file.yaml"
```

Comment:
```suggestion
  # openstack_config_file_path: <PATH_TO_YAML_FILE>
```

Reply:
let's follow the configuration file norm https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#configuration-file